### DISCUSSION TITLE
Nesting with select

### DISCUSSION THREAD

**Discussion by Rahmo** [prisma/prisma#5366] [Q&A] [answered]
nesting selection? it may be a simple Q - I want the data looks in certain way I could manipulate them with map I just wanna make sure its the right way... 

my current results from resolver : 
```
schools:{
 {
        "type": "",
        "branch": "NA"
      },
      {
        "type": "Language",
        "branch": "NA"
      },
      {
        "type": "Institute",
        "branch": "UAE"
      },
}
```
what I want as a results is like 
```
{
type : ["","Language","institue"]
branch :[NA,"UAE"]
}
```
my schema 
 
```
	type group {
	        kind: String
		branch: String
	}
	 type Query {
            schools: [group]
    }
```
resolver
```
schools: async (parent, args,context) => {
   const navigations = await context.prisma.schools.findMany({
      select: {
         kind: true,
         branch : true,
      },
      distinct: ["kind", "branch"],
   })
   console.log(schools)
   return schools
},
```

I am executing it like below
```
schools{
    type,
    branches
  }
```

any thoughts of how to do this nesting from Prisma and graphQL without having to manipulate data with map and flat maps ?

**Comment by ryands17** [comment, marked as answer]
@Rahmo 👋 
I don't think this is possible directly as Prisma will give you the exact structure as you have in your models defined in `schema.prisma`. You would need to manipulate this in your application logic itself in this case.

**Reply by Rahmo** [reply]
yeah - thank you my data table has been bad... thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
